SECP-service/src/main/java/com/visucius/secp/resources/UserResource.java

```diff
@@ -98,4 +98,17 @@ public Response verifyDevice(@Auth @PathParam("name") String deviceName, @PathPa
     public Response getUsersPublicKeys(@Auth @PathParam("id") long id) {
         return userController.getUsersPublicKeys(id);
     }
+
+    @GET
+    @UnitOfWork
+    public Response getAllUsers() {
```

Comment:
Should have the @Auth tag here

Reply:
since there are no parameters. You can't put the @Auth tag. The @RolesAllowed tag already verifies if the user is authorized.

Reply:
Good question